github/MainClass.pyi

```diff
@@ -46,10 +48,10 @@ class Github:
     def __set_per_page(self, value: int) -> None: ...
     def create_from_raw_data(
         self,
-        klass: Union[Type[Repository], Type[NamedUser]],
+        klass: Type[TGithubObject],
         raw_data: Dict[str, Any],
         headers: Dict[str, Union[str, int]] = ...,
-    ) -> Union[Repository, NamedUser]: ...
+    ) -> TGithubObject: ...
     def dump(self, obj: Repository, file: BytesIO, protocol: int = ...) -> None: ...
```

Comment:
Should this also change from Repository to TGithibObject as well?

Reply:
I'm not using it in my code so didn't notice, but I suppose so! Update pushed.